assets/section-main-product.css

```diff
@@ -23,7 +23,10 @@
 }
 
 @media screen and (min-width: 750px) {
-  .product__info-container--sticky {
+  .product--thumbnail .product__media-gallery,
+  .product--thumbnail_slider .product__media-gallery,
+  .product--stacked .product__info-container--sticky {
+    display: block;
     position: sticky;
```

Comment:
One thing I notice here with position sticky on the media gallery is that it makes it a bit annoying to get to see the video: [video](https://screenshot.click/21-12-dt0hk-74tj7.mp4)

Reply:
Hum, this is interesting. A few things comes to mind:
- This is not only impacting videos but any media in a landscape ratio
- This is not something that was concerning with the stacked layout because we had more vertical visuals to view
- Since merchants can deactivate the `Enable sticky header` setting and the other one, I am not too concerned. That behaviour is not meant for all instances and this is the reason why the settings exist.
- Our defaults set merchants for success if they remain with the `Stacked` layout.